server/src/internalClusterTest/java/org/elasticsearch/index/shard/SearchIdleIT.java

```diff
@@ -182,4 +200,27 @@ private void ensureNoPendingScheduledRefresh(ThreadPool threadPool) {
         barrier.arriveAndAwaitAdvance();
     }
 
+    public void testSearchIdleStats() throws InterruptedException {
```

Comment:
Maybe also add a yaml test to `rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/indices.stats` dir?
Just to verify that the new fields are included in the stats response

Reply:
I was thinking about that even if the stats api sued at shard level returns a lot of data that I can't actually check with `match`. Also for search idle I can't check the values in the test...just that the fields exist.